receiver/splunksearchapireceiver/testdata/logs/testPollJobStatus/input.xml

```diff
@@ -0,0 +1,212 @@
+<entry
+       xmlns="http://www.w3.org/2005/Atom"
+       xmlns:s="http://dev.splunk.com/ns/rest"
+       xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">
+  <title>search index</title>
+  <id>https://localhost:8089/services/search/jobs/mysearch_02151949</id>
+  <updated>2011-07-07T20:49:58.000-07:00</updated>
+  <link href="/services/search/jobs/mysearch_02151949" rel="alternate"/>
+  <published>2011-07-07T20:49:57.000-07:00</published>
+  <link href="/services/search/jobs/mysearch_02151949/search.log" rel="search.log"/>
+  <link href="/services/search/jobs/mysearch_02151949/events" rel="events"/>
+  <link href="/services/search/jobs/mysearch_02151949/results" rel="results"/>
+  <link href="/services/search/jobs/mysearch_02151949/results_preview" rel="results_preview"/>
+  <link href="/services/search/jobs/mysearch_02151949/timeline" rel="timeline"/>
+  <link href="/services/search/jobs/mysearch_02151949/summary" rel="summary"/>
+  <link href="/services/search/jobs/mysearch_02151949/control" rel="control"/>
+  <author>
+    <name>admin</name>
+  </author>
+  <content type="text/xml">
+    <s:dict>
+      <s:key name="cursorTime">1969-12-31T16:00:00.000-08:00</s:key>
+      <s:key name="delegate"></s:key>
+      <s:key name="diskUsage">2174976</s:key>
+      <s:key name="dispatchState">DONE</s:key>
```

Comment:
I think having a dispatchState as not done will be another important test for any future changes.